### PR TITLE
Audio player with liner notes: use BookReader v5

### PR DESCRIPTION
**Pull request by iisa**
**Description**

Updates IA BookReader DOM to use web component

**Technical**

- storybook pulls in latest script tags that will activate the web component

**Testing**
- pull down branch -> go to: /packages/ia-components
- open `~/.storybook/preview-head.html` and point scripts to `www-isa2.archive.org` instead of `archive.org` to fetch latest bookreader & archive.org changes
- `yarn install && yarn run storybook`
- storybook runs -> go to theater demos -> "Audio with liner notes"
- toggle `Liner Notes` tab -> BookReader with new side panels are now being used
- toggle fullscreen -> bookreader toggles in & out of fullscreen

**Evidence**

bookreader shows liner notes in storybook demo & on [staging app](https://www-isa2.archive.org/details/cd_hanna-barbera-cartoon-sound-fx_william-hanna-joseph-barbera )

ezgif com-gif-maker (9)

### CODE REVIEW

File: packages/ia-components/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "@babel/plugin-transform-runtime": "^7.5.5",
     "@babel/preset-env": "^7.13.10",
     "@babel/runtime": "^7.5.5",
-    "@internetarchive/bookreader": "^4.37.0",
+    "@internetarchive/bookreader": "^5.0.0-0",
```
Comment (cdrini) [inline packages/ia-components/package.json:26]: We want this one pinned; we'll likely be making breaking changes to this and don't want to it to autoupdate. Also, I messed up, and my experimental `5.0.0-beta-18ceefd` matches over `^5.0.0-0` :( Need to find a fix for that.

```suggestion
    "@internetarchive/bookreader": "5.0.0-1",
```

Comment (iisa) [reply packages/ia-components/package.json:26]: hi drini thanks for the suggestion. this must stay in lock-step with latest bookreader, so it should be ok to use caret ^_^

File: packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx [hunk 2]
```diff
@@ -431,13 +436,18 @@ AudioPlayerWithYoutubeSpotify.defaultProps = {
   jwplayerPlaylist: null,
   playFullIAAudio: false,
   linerNotes: null,
+  userSignedIn: false,
+  albumMetadataStr: '',
+  albumMetadata: null,
```
Comment (nsharma123) [inline packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx:441]: We know there are two new properties which is **albumMetadataStr** and **userSignedIn**, Just curious setting albumMetadata to null won't make anything wrong?

File: packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js [hunk 3]
```diff
@@ -115,8 +115,8 @@ class DataHydrator extends Component {
           <h4>Archive </h4>
           <table>
             <thead>
-              <td>Collection</td>
-              <td>Identifier</td>
+              <tr>Collection</tr>
+              <tr>Identifier</tr>
```
Comment (nsharma123) [inline packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js:119]: I would write it with one more TR and TH.
`  <thead>
    <tr>
      <th>Month</th>
      <th>Savings</th>
    </tr>
  </thead>
`

File: packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx (hunk 2 above)
Comment (iisa) [reply packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx:441]: hi neeraj, good question, no it doesn't actually affect anything.  all of these properties get hydrated at the same time so we should be good.  the linter was barking that `albumMetadata` didn't have a prop so i added one.

File: packages/ia-components/components/audio-players/audio-player.less [hunk 4]
```diff
@@ -16,6 +16,7 @@
   }
 
   .content-window {
+    height: 440px;
```
Comment (bfalling) [inline packages/ia-components/components/audio-players/audio-player.less:19]: SUGGESTION/QUESTION: Significance of this magic number? I see it repeated below. Should it be a variable?

File: packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js (hunk 3 above)
Comment (bfalling) [reply packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js:119]: Was this changed? (I don't see the update.)

Comment (iisa) [reply packages/ia-components/components/theatres/with-youtube-spotify/audio-with-youtube-spotify.stories.js:119]: not yet as this is the demo. scripts need to be updated as soon I release bookreader so demo can properly work and I will update then 👍

**Pull request merged**